src/Microsoft.CSharp/src/Microsoft.CSharp.csproj

```diff
@@ -15,6 +15,10 @@
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
   </PropertyGroup>
+  <PropertyGroup Label="MultilingualAppToolkit">
+    <MultilingualAppToolkitVersion>4.0</MultilingualAppToolkitVersion>
+    <MultilingualFallbackLanguage>en-US</MultilingualFallbackLanguage>
+  </PropertyGroup>
```

Comment:
I second Stephen here. in build tools we can have the defaults for all projects and each project can customize it if need to.